prescription.sh

```diff
@@ -57,7 +57,8 @@ function generateYML () {
         --asset.id=*) asset_id="${i#*=}" ;;
         --slack.channel.id=*) slack_channel_id="${i#*=}" ;;    #slack
         --slack.token=*) slack_token="${i#*=}" ;;
-        --jira.project.name=*) jira_project_name="${i#*=}" ;;    #jira
+        --jira.enable=*) enable_jira="${i#*=}" ;;    #jira
```

Comment:
Do we want to replace `<<ENABLE_JIRA>>` placeholder with `enable_jira` variable below where we are doing the same for other placeholders using `sed`  from line 178

Reply:
added